demo/demo.js

```diff
@@ -1,3 +1,5 @@
+/*eslint-disable */
+
 // create dataset
 var data = [];
```

Comment:
I kept voluntarily the demo in ES5 to widen our target. It is then ready to copy/paste without having to configure any transpilers.